packages/build/src/npm-packages/bump.ts

```diff
@@ -13,27 +12,40 @@ export function bumpNpmPackages(
   }
 
   console.info(`mongosh: Bumping package versions to ${version}`);
-  spawnSyncFn(
-    LERNA_BIN,
-    [
-      'version',
-      version,
-      '--no-changelog',
-      '--no-push',
-      '--exact',
-      '--no-git-tag-version',
-      '--force-publish',
-      '--yes',
-    ],
-    {
-      stdio: 'inherit',
-      cwd: PROJECT_ROOT,
-      encoding: 'utf8',
+  const monorepoRootPath = path.resolve(__dirname, '..', '..', '..', '..');
+  const packages = await getPackagesInTopologicalOrder(monorepoRootPath);
+
+  const locations = [monorepoRootPath, ...packages.map((p) => p.location)];
+
+  for (const location of locations) {
+    const packageJsonPath = path.join(location, 'package.json');
+    console.log({ packageJsonPath });
+    const packageJson = JSON.parse(await fs.readFile(packageJsonPath, 'utf8'));
+
+    packageJson.version = version;
+    for (const grouping of [
+      'dependencies',
+      'devDependencies',
+      'optionalDependencies',
+      'peerDependencies',
+    ]) {
+      if (!packageJson[grouping]) {
+        continue;
+      }
+
+      for (const [name, currentVersion] of Object.entries(
+        packageJson[grouping]
+      )) {
+        if (currentVersion !== PLACEHOLDER_VERSION) {
```

Comment:
Wouldn’t this be a more accurate check? I get that in practice we’re not going to include packages in our dependencies that happen to use the placeholder as their actual version, but what we’re really going for here semantically is bumping the packages that are also _from this monorepo_, right?

```suggestion
        if (!packages.map(p => p.name).includes(name)) {
```

(This is optional but fwiw, the test file here needs to be updated as well)

Reply:
Yeah I have to write tests. Just wanted to see if this fixes the CI failure in the meantime.